track_health.rb

```diff
@@ -1,7 +1,40 @@
 require_relative 'encounter'
+require 'json'
 
-encounter_file_location = 'data/encounters/encounter-1.json'
-encounter = Encounter.new(encounter_file_location)
+ENCOUNTERS_DIRECTORY = 'data/encounters'.freeze
 
-puts "Encounter Name: #{encounter.name}"
-puts "Monster Names: #{encounter.monster_names.join(', ')}"
+def put_encounter_name(encounter_path)
+  encounter_being_checked = JSON.parse(File.read("#{encounter_path}"))
```

Comment:
Let's put this on its own line and assign it to a variable:
`File.read("#{encounter_path}"`

Reply:
You can just use `.to_s` if you're not sure that this is a string. No need to escape quotes with `"#{}"` if you aren't concatenating anything.